core/src/main/java/org/mule/runtime/core/api/routing/RoutePathNotFoundException.java

```diff
@@ -21,19 +20,19 @@
    */
   private static final long serialVersionUID = -8481434966594513066L;
 
-  public RoutePathNotFoundException(MuleEvent event, MessageProcessor target) {
-    super(event, target);
+  public RoutePathNotFoundException(MessageProcessor target) {
```

Comment:
Done: MULE-10519. I kept the message processor in this case not because of the "failingProcessor" stuff but because it is used to check the route.